src/constants/index.ts

```diff
@@ -142,7 +142,7 @@ export const COINGECKO_API_URL = 'https://api.coingecko.com/api/v3'
 export const KNC_COINGECKO_ID = 'kyber-network-crystal'
 
 export const ETHER_ADDRESS = '0xEeeeeEeeeEeEeeEeEeEeeEEEeeeeEeeeeeeeEEeE'
-export const ETHER_ADDRESS_SOLANA = 'eeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee'
+export const ETHER_ADDRESS_SOLANA = 'So11111111111111111111111111111111111111111'
```

Comment:
where did you get this?

Reply:
Backend sees it as a native token.
curently, FE only use ETHER_ADDRESS_SOLANA for price alert. 
price alert is ok with So11111111111111111111111111111111111111111 is present native.  
=> we will use this address for native sol

Reply:
Can you give an example of API result? I dont get it where this address came from

Reply:
https://ks-setting.dev.kyberengineering.io/api/v1/tokens?query=sol&chainIds=101&page=1&pageSize=20
<img width="1126" alt="image" src="https://github.com/KyberNetwork/kyberswap-interface/assets/33005392/d5a48ccc-440d-4e79-b3d0-f404707e2772">

Reply:
It's not on prod yet?

Reply:
yep, will add to admin when this pr release